slam-libraries/Makefile

```diff
@@ -1,7 +1,14 @@
 BUILD_CHANNEL?=local
 
 bufsetup:
-	sudo apt-get install -y protobuf-compiler-grpc libgrpc-dev libgrpc++-dev || brew install grpc openssl --quiet
+ifeq ("Darwin", "$(shell uname -s)")
+	sudo rm /usr/local/bin/go || true
```

Comment:
This is necessary to unlink go version 1.17 in CI, but I don't want to have to do this when running locally. Any ideas for how to do this differently?